### PR TITLE
Add tests for subwf example

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

It adds missing parsing tests for one example.This will help prevent code regression

Thanks,

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ install:
   - pip install -r ../requirements.txt
   - sudo apt-get install coreutils libxml2-utils
 script:
-  - pre-commit run --all-files
   - ./run-all-conversions
+  - pre-commit run --all-files
```
Comment (potiuk) [inline .travis.yml]: I assume you want to run tests on all the generated files as well ? Good idea :).

File: oozie-to-airflow/mappers/subworkflow_mapper.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def _parse_oozie_node(self):
         app_path = el_utils.replace_el_with_var(app_path, params=self.params, quote=False)
         # TODO: hacky: we should calculate it deriving from input_directory_path and comparing app-path
         # TODO: but for now we assume app is in "examples"
-        app_path = os.path.join(EXAMPLES_PATH, app_path.split("examples/")[1])
+        app_path = os.path.join(EXAMPLES_PATH, app_path.rsplit("/", 1)[1])
```
Comment (potiuk) [inline oozie-to-airflow/mappers/subworkflow_mapper.py]: Nice one :).

File: oozie-to-airflow/tests/converter/test_oozie_parser.py [hunk 3]
```diff
@@ -473,6 +473,7 @@ class TestOozieExamples(unittest.TestCase):
                     params={"hostname": "AAAA@BBB", "nameNode": "hdfs://"},
                 ),
             ),
+            (WorkflowTestCase(name="subwf", node_names={"subworkflow_node"}, relations=set(), params={}),),
```
Comment (potiuk) [inline oozie-to-airflow/tests/converter/test_oozie_parser.py]: Cool

File: .travis.yml (hunk 1 above)
Comment (mik-laj) [reply .travis.yml]: Yes. Subworkflow converter requires a properties..

**Pull request merged**